src/board_controller/brainflow_boards.cpp

```diff
@@ -710,14 +710,14 @@ BrainFlowBoards::BrainFlowBoards()
     {
         {"name", "BrainAlive"},
         {"sampling_rate", 250},
-        {"timestamp_channel", 15},
-        {"marker_channel", 16},
-        {"package_num_channel", 0},
+        {"package_num_channel", 14},
+        {"marker_channel", 15},
+        {"timestamp_channel", 16},
         {"num_rows", 17},
-        {"eeg_channels", {1, 2, 3, 4, 5, 6, 7, 8}},
+        {"eeg_channels", {0, 1, 2, 3, 4, 5, 6, 7}},
```

Comment:
why did you remove package_num_channel from position 0? Its there for all boards, and lets keep it consistent